app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,10 @@
             </intent-filter>
         </activity>
 
+        <service
```

Comment:
Hmm before a service wasn't obligatory. But I guess this is needed now :(.

Reply:
This service is (I believe) an example of how we can get notifications of beacons entering / exiting.

We can probably put this service into the Harald SDK itself, and modify the behaviour of what to do when entering a beacon / exiting a beacon by default.

Right, @nielsdelporte ?

Reply:
That's right, @SanderVH .
Job services should be protected with this permission, but this service (and permission) is added in the sample app and not part of the iBeaconScanner library.